javascript/ql/src/semmle/javascript/dataflow/internal/FlowSteps.qll

```diff
@@ -97,11 +97,33 @@ private module CachedSteps {
     f = cap.getContainer()
   }
 
+  /**
+   * Holds if the method invoked by `invoke` resolved to a member named `name` in `cls`
+   * or one of its super classes.
+   */
+  cached
+  predicate callResolvesToClass(DataFlow::InvokeNode invoke, DataFlow::ClassNode cls, string name) {
```

Comment:
The name of this predicate is a bit strange; wouldn't something like `callResolvesToMember` (or `callResolvesToClassMember`) be more appropriate?

Reply:
Completely agree